### PR TITLE
Update routerAddresses.ts: add solana drift protocol

### PR DESCRIPTION
**Pull request by 0xbigz**
add solana's drift protocol program ids to routerAddresses.ts

(a follow up on https://github.com/DefiLlama/DefiLlama-Adapters/issues/6356)
todo: what should be in "id" field?

### CODE REVIEW

**Comment by dtmkeng** [comment]
@0xngmi

**Comment by 0xbigz** [comment]
@dtmkeng bumping here :)

i pull/resolved conflict with master

File: users/routers/routerAddresses.ts [hunk 1]
```diff
@@ -2676,5 +2676,15 @@ export default ([
             ]
         }
     }, 
+    {
+        "id":"3388", // todo?
```
Comment (0xngmi) [inline users/routers/routerAddresses.ts]: why is the id set to 3388? drift's id is 970

the protocol with id 3388 is sanctum

**Comment by 0xngmi** [comment]
merged, however we dont have infra support for solana yet so data wont appear right away

**Pull request merged**